backend/src/test/java/com/github/chuettenrauch/mixifyapi/unit/mixtape_user/service/MixtapeUserServiceTest.java

```diff
@@ -37,6 +38,41 @@ void findAllForUser_whenCalled_thenDelegateToMixtapeUserRepository() {
         verify(mixtapeUserRepository).findAllByUser(user);
     }
 
+    @Test
+    void findByUserAndMixtape_whenFound_thenReturn() {
+        // given
+        User user = new User();
+        Mixtape mixtape = new Mixtape();
+
+        MixtapeUser expected = new MixtapeUser(null, user, mixtape);
+
+        MixtapeUserRepository mixtapeUserRepository = mock(MixtapeUserRepository.class);
+        when(mixtapeUserRepository.findByUserAndMixtape(user, mixtape)).thenReturn(Optional.of(expected));
+
+        // when
+        MixtapeUserService sut = new MixtapeUserService(mixtapeUserRepository);
+        MixtapeUser actual = sut.findByUserAndMixtape(user, mixtape);
+
+        // then
+        assertEquals(expected, actual);
+        verify(mixtapeUserRepository).findByUserAndMixtape(user, mixtape);
+    }
+
```

Comment:
Testest du eigentlich noch mit konkreten Daten, statt mit leeren Objekten (User, Mixtape)? Vermutlich in den jeweiligen Testklassen für User/Mixtape

Reply:
Eigentlich fast nur noch bei den Integrationtests. Bei den Unittests, nur wenn es relevant ist.